.github/workflows/update_tgs_dmapi.yml

```diff
@@ -0,0 +1,44 @@
+name: Update TGS DMAPI
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  update-dmapi:
+    runs-on: ubuntu-latest
+    name: Update the TGS DMAPI
+    steps:
+    - name: Clone
+      uses: actions/checkout@v2
+
+    - name: Branch
+      run: |
+        git branch -f tgs-dmapi-update
+        git checkout tgs-dmapi-update
+        git reset --hard master
+
+    - name: Apply DMAPI update
+      uses: tgstation/tgs-dmapi-updater@v2
+      with:
+        header-path: 'code/__DEFINES/tgs.dm'
+        library-path: 'code/modules/tgs'
+
+    - name: Commit and Push
+      run: |
+        git config user.name tgstation-server
+        git config user.email tgstation-server@users.noreply.github.com
+        git commit -a -m 'Update TGS DMAPI'
+        git push -f -u origin tgs-dmapi-update
+
+    - name: Create Pull Request
+      uses: repo-sync/pull-request@v2
+      with:
+        source_branch: "tgs-dmapi-update"
+        destination_branch: "master"
+        pr_title: "Automatic TGS DMAPI Update"
+        pr_body: "This pull request updates the TGS DMAPI to the latest version. Please note any breaking or unimplemented changes before merging."
+        pr_label: "Tools"
+        pr_allow_empty: false
+        github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
```suggestion
        #github_token: ${{ secrets.GITHUB_TOKEN }} Use this instead if you have unprotected branches
        github_token: ${{ secrets.CL_TOKEN }}
```

Reply:
This shouldn't be needed though, unless you have a wildcard for your branch protection?

Reply:
Was doing it mostly for consistency sake. You are correct that it's not strictly required, however.